app/controllers/users_controller.rb

```diff
@@ -0,0 +1,70 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
rename to `authorize_user!`

A method name should be an action.

Reply:
Naming came from the book, will modify it accordingly to be more expressive. ✓